packages/radix-ui-themes/src/components/checkbox.tsx

```diff
@@ -4,15 +4,17 @@ import * as React from 'react';
 import classNames from 'classnames';
 import * as CheckboxPrimitive from '@radix-ui/react-checkbox';
 import { checkboxPropDefs } from './checkbox.props.js';
-import { extractProps, marginPropDefs } from '../helpers/index.js';
-import { ThickCheckIcon } from '../icons.js';
+import { extractProps } from '../helpers/index.js';
+import { marginPropDefs } from '../props/index.js';
```

Comment:
Would it make sense to always combine imports from the barrel file (equivalent of `@radix-ui/themes/props` and `@radix-ui/themes/helpers`?

So here for example:

```tsx
import { checkboxPropDefs, marginPropDefs } from '../props/index.js';
```

Reply:
I am slightly biased towards keeping the component portable with its props, but not a strong preference if you prefer cleaner imports. E.g. if we wouldn't have to export the prop defs for the docs, I wouldn't put them into prop index at all.

Reply:
Let's keep as is for now then.